lib/log_parser.rb

```diff
@@ -1,15 +1,28 @@
 class LogParser
   attr_accessor :file_path
+  require "json"
     
   def initialize(file_path)
     @file_path = file_path
     raise "File not found" unless File.exists?(@file_path)
-    @file = File.open(@file_path)       
+    @file = File.open(@file_path)
+    @line_count = 0    
   end
     
   def read_first_line
     first_line = @file.readline
     @file.close
     first_line        
-  end       
+  end
+
+  def parse_file    
+    parse_json = JSON.generate({@file_path => {"lines" => line_counter}})
+    @file.close
+    parse_json
+  end
+
+  private
+  def line_counter    
+    @line_count = @file.readlines.length
```

Comment:
@ivopozzani não há necessidade de atribuir o resultado a uma variável de instância, vc pode simplesmente retornar o valor aqui

```suggestion
    @file.readlines.length
```